src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/steps/base-step/base-step.component.ts

```diff
@@ -27,30 +36,51 @@ export class BaseStepComponent extends BaseComponent implements OnInit {
     @Input() public sectionStepRadioData: RadioControlType[];
     @Input() public sectionFormData: JsonFormData;
     @Input() public defaultKind: string;
-    @Input() public groupName: string;
+    @Input() public groupName: SetPollingSourceSection;
     @Input() public title: string;
```

Comment:
I think that `title` can be directly derived from `groupName`

Reply:
Fixed